src/modules/methods.js

```diff
@@ -0,0 +1,157 @@
+// import from src modules folder
+
+import DataList from './datalist.js';
+
+// get listed inputs from local storage
+
+export default class MainMethod {
+  static getToDoListFromLocalStorage = () => {
+    let toDoLists;
+
+    if (JSON.parse(localStorage.getItem('LocalDataList')) === null) {
+      toDoLists = [];
+    } else {
+      toDoLists = JSON.parse(localStorage.getItem('LocalDataList'));
+    }
+    return toDoLists;
+  };
+
+  // add listed inputs to the local storage
+  static addListToStorage = (toDoLists) => {
+    const item = JSON.stringify(toDoLists);
+    localStorage.setItem('LocalDataList', item);
+  };
+
+  // index list inputs by number
+  static newIndexNum = (toDoLists) => {
+    toDoLists.forEach((item, i) => {
+      item.index = i + 1;
+    });
+  }
+
+  // delete from local storage
+    static deleteListData = (id) => {
+      let toDoLists = this.getToDoListFromLocalStorage();
+      const ListItemToDelete = toDoLists[id];
+
+      toDoLists = toDoLists.filter((item) => item !== ListItemToDelete);
+
+      this.newIndexNum(toDoLists);
+      this.addListToStorage(toDoLists);
+    };
```

Comment:
Good day dear code review team, i've recently reached out to @Alfredbis29 regarding the required changes noted above, during our zoom call I had the chance to share my screen and show him the required changes were already implemented and functional as close as possible for this projects milestone requirements, we assumed it might have been a browser error or the webpackage was not loaded fully when he had the chance to test the project, and so I thought taking screen shots of the requirements being present and working in my project already would not do it justice so I took the liberty of creating a [demonstration video linked here](https://drive.google.com/file/d/1tDGbtNbsjyLqoyPulFd6FFlGfnTGuyPr/view?usp=sharing) which shows me explaining that the index order works on both google chrome and Microsoft edge

### issue 1
- the to do list didn't seem to be indexing the listed tasks in the correct order when one of the listed tasks was deleted or when the page was reloaded

### video explanation for issue 1
( **in the video I show how i listed items in the to-do list, and how they are re-ordered  with the index when deleted and remained in that order when the page was refreshed**)

### issue 2
- it seemed as if there was no functionality to edit the already listed tasks, and they would disappear when the page was reloaded

### video explanation for issue 2
(**in the video I add a list of numbers { 1,2,3,4 } to the todo list, refresh the page a few times, and then i spelled out the numbers in words while refreshing the page after i add a new word number**)